resinos-in-container.sh

```diff
@@ -78,6 +84,9 @@ while [[ $# -ge 1 ]]; do
 		--clean-volumes)
 			clean_volumes=yes
 			;;
+		--assume-root)
```

Comment:
How about checking if the user id is 0 instead of using a flag?
i.e.
```
if [ "$(id -u)" = "0" ]; then
  sudo=""
fi
```
and then no need for --assume-root to exist.

Reply:
Very good point.